.env.example

```diff
@@ -9,3 +9,11 @@ OPENDAL_S3_BUCKET=<bucket>
 OPENDAL_S3_ENDPOINT=<endpoint>
 OPENDAL_S3_ACCESS_KEY_ID=<access_key_id>
 OPENDAL_S3_SECRET_ACCESS_KEY=<secret_access_key>
+# azblob
+OPENDAL_AZBLOB_TEST=false
+OPENDAL_AZBLOB_ROOT=/path/to/dir
+OPENDAL_AZBLOB_BUCKET=<bucket>
+//endpoint look like that blob.core.windows.net
+OPENDAL_AZBLOB_ENDPOINT=<endpoint>
+OPENDAL_AZBLOB_ACCESS_NAME=<access_name>
```

Comment:
Please keep the same naming style within azblob. That means `account_name` and `account_key`.